UI/MainMenu/SaveGameElement.gd

```diff
@@ -37,23 +38,14 @@ func setSaveFile(path):
 func _on_LoadButton_pressed():
 	emit_signal("onLoadButtonPressed", saveFile)
 
-
 func _on_DeleteButton_pressed():
 	emit_signal("onDeleteButtonPressed", saveFile)
+	queue_free() #Keep it simple!!! If you havn't any steps what can revert action, free node!
```

Comment:
Why should it delete itself?

Reply:
Well... If you delete save, UI node don't needed, right?